tracing-futures/CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# 0.2.0 (Nov XX, 2019)
+
+### Changed
+
+- the default `Future` implementation comes from the `std-future` feature.
```

Comment:
Can this make it clearer that this was the breaking change that caused the version bump? Thanks!
```suggestion
- **Breaking Change**: the default `Future` implementation comes from the `std-future` feature.
```